service-node/src/main/scala/org/enterprisedlt/fabric/service/node/cryptography/FileBasedCryptoManager.scala

```diff
@@ -24,44 +24,50 @@ import org.slf4j.LoggerFactory
   */
 class FileBasedCryptoManager(
     organizationConfig: OrganizationConfig,
-    rootDir: String
+    rootDir: String,
+    adminPassword: String
 ) extends CryptoManager {
     private val logger = LoggerFactory.getLogger(this.getClass)
     private val orgFullName = s"${organizationConfig.name}.${organizationConfig.domain}"
+    private val cryptoDir = s"$rootDir/crypto"
     //
     // Initialization
     //
     val notBefore: Date = new Date()
     val notAfter: Date = Util.futureDate(Util.parsePeriod(organizationConfig.certificateDuration))
-    val orgCryptoMaterial: OrganizationCryptoMaterial = if (!new File(s"$rootDir/ca").exists()) {
+    val orgCryptoMaterial: OrganizationCryptoMaterial = if (!new File(s"$cryptoDir/ca").exists()) {
         logger.info(s"Generating crypto for $orgFullName...")
         val orgCryptoMaterial = FabricCryptoMaterial.generateOrgCrypto(
-            organizationConfig, orgFullName, rootDir,
+            organizationConfig, orgFullName, cryptoDir,
             notBefore, notAfter
         )
         logger.info(s"Generated crypto for $orgFullName.")
         orgCryptoMaterial
     } else {
         logger.info(s"Crypto for $orgFullName already exist.")
         OrganizationCryptoMaterial(
-            loadCertAndKey(s"$rootDir/ca/ca"),
-            loadCertAndKey(s"$rootDir/tlsca/tlsca"),
-            loadCertAndKey(s"$rootDir/users/admin/admin")
+            loadCertAndKey(s"$cryptoDir/ca/ca"),
+            loadCertAndKey(s"$cryptoDir/tlsca/tlsca"),
+            loadCertAndKey(s"$cryptoDir/users/admin/admin")
         )
     }
 
+    logger.info(s"Obtaining user key for admin ...")
```

Comment:
Pls adjust comment to something more appropriate

Reply:
adjusted.